components/nimbus/src/nimbus_client.rs

```diff
@@ -85,6 +86,10 @@ impl NimbusClient {
             app_context,
             database_cache: Default::default(),
             db_path: db_path.into(),
+            // With this being default, i.e. empty, Nimbus doesn't support coenrolling ids.
+            // Once the API is connected with Swift/Kotlin it will.
+            // This is the subject of https://mozilla-hub.atlassian.net/browse/EXP-3623
+            coenrolling_feature_ids: Default::default(),
```

Comment:
Okay so the idea here is the Swift/Kotlin generated code will send in the coenrolling feature ids when instantiating Nimbus?

Reply:
The `NimbusBuilder` will mediate: grabbing the generated list of ids from the generated code, and poke it into the `NimbusClient` at construction time.